api/src/main/java/org/pmiops/workbench/config/AppEngineConfig.java

```diff
@@ -0,0 +1,16 @@
+package org.pmiops.workbench.config;
+
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.http.HttpTransport;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppEngineConfig {
+
+    @Bean
+    HttpTransport httpTransport() {
```

Comment:
The HttpTransport bean is one of the key App Engine specific dependencies we inject. I'd originally tried putting this bean within WebMvcConfig, but that was causing a cyclic bean dependency (since Spring tries to load all beans for a given class at the same time). Pulling it out into its own file here resolved the issue.